gnucash/import-export/csv-exp/csv-transaction-export-line.cpp

```diff
@@ -379,26 +384,6 @@ CsvTransactionExportLine::make_gdpdu_trans_line(Split *split)
         get_amount(split, t_void, false)};
 }
 
-StringVec
-CsvTransactionExportLine::make_simple_trans_split_line(Split *split)
-{
-    auto t_void{xaccTransGetVoidStatus(m_transaction)};
-    gnc_numeric amount = xaccSplitGetAmount(split);
-    bool pos = gnc_numeric_positive_p(amount);
-    return {
-        get_date(m_transaction),
-        pos ? xaccAccountGetName(m_base_split_account) : get_account_name(split, true),
-        get_number(m_transaction),
-        get_description(m_transaction),
-        pos ? get_account_name(split, true) : xaccAccountGetName(m_base_split_account),
-        get_reconcile(split),
-        get_amount(split, t_void, true),
-        get_amount(split, t_void, false),
-        get_value(split, t_void, true),
-        get_value(split, t_void, false),
-        get_rate(split, t_void)};
-}
-
```

Comment:
This should be removed from the second commit instead of adding it there and throwing it away later.